src/test/java/edu/pdx/cs/IOHanderTest.java

```diff
@@ -0,0 +1,19 @@
+package edu.pdx.cs;
+
+import net.sourceforge.argparse4j.inf.ArgumentParserException;
+import net.sourceforge.argparse4j.inf.Namespace;
+import org.junit.Test;
+
+import java.util.Collection;
+
+public class IOHanderTest {
+
+    IOHandler ioHandler = new IOHandler();
+
+    @Test
+    public void ioTest() throws ArgumentParserException {
+        Namespace namespace = ioHandler.parser.parseArgs(new String[] { "--login", "username", "password"});
+        Collection command = namespace.getAttrs().values();

```

Comment:
Is there an assertion in this test? Could construct an expected `Namespace` and assert equality, for example.